src/Engines/MeilisearchEngine.php

```diff
@@ -378,7 +372,13 @@ public function deleteIndex($name)
      */
     public function deleteAllIndexes()
     {
-        return $this->meilisearch->deleteAllIndexes();
+        $tasks = [];
+        $indexes = $this->meilisearch->getIndexes();
```

Comment:
@mmachatschek, just to inform you, this does not remove all the indexes. It only removes the first page of indexes, which will probably be 20.

Since this method did not remove all the indexes correctly and PHP would be the only one with it, I opted to remove it.
A correct implementation in my POV is to correctly iterate over all the pages of the response :)

Reply:
I thought so, just read through the changelog of the SDK. at least the implementation of the method here on scout side behaves the same, although its wrong. I'll update the code to address this issue

Reply:
@brunoocasali the PR is ready for another round of review